README.md

```diff
@@ -1,69 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
```

Comment:
- [ ] You should update the title of the project to the new project. e.g. Portfolio, My Portfolio, etc